src/adapters/inbound/rest/v1/controllers/cliente/cliente.controller.ts

```diff
@@ -23,6 +24,9 @@ import { BadRequestError } from '../../../helpers/swagger/status-codes/bad_reque
 import { NotFoundError } from '../../../helpers/swagger/status-codes/not_found.swagger';
 import { ConflictError } from '../../../helpers/swagger/status-codes/conflict.swagger';
 
+import {validate as uuidValidate} from 'uuid'
```

Comment:
CI apresento erro:

![image](https://github.com/Grupo-G03-4SOAT-FIAP/rms-backend-fase01/assets/30847848/96b95256-acc1-402e-bca8-797251c349fb)

Reply:
Algum motivo especial para usar o "as" no import?

Reply:
se for igual o python o `as` serve como um apelido, quando uma função não tem o nome claro e pode prejudicar a analise e dar conflito com outros metodos/funções é recomendado usar o `as`.

Reply:
Resolvido!

As em JS/TS tem a mesma funcinalidade em python, apelidar uma lib/módulo para facilitar a produção e leitura do código, mas já foi removido as funções relacionadas ao UUID e acrescentado o ParseUUID do Nest